.github/SECURITY.md

```diff
@@ -1,13 +1,21 @@
 # Security Policy
 
-If you discover a security vulnerability in this project, please
-report it privately. **Do not disclose it as a public issue.**
+If you discover a security vulnerability in this project, especially a highly dangerous one, you can either report it privately or disclose it as a public issue, so that more people are informed of the situation.
```

Comment:
This goes against literally all security best practices, which leaves the question: why are you suggesting this?